src/modules/dataFetching/components/GenericTableWithData.tsx

```diff
@@ -251,14 +253,16 @@ const GenericTableWithData = <
         setPage(0);
       },
       count: nodesCount,
+      labelRowsPerPage,
```

Comment:
Maybe we can use the existing `recordType` or `paginationItemName` props so that we don't end up with diverging implementations of the `per page:` part.

This should work:
```suggestion
      labelRowsPerPage: `${pluralize(startCase(paginationItemName || 'Row'))} per page:`,,
```

It think this basically matches what we do for the "displaying 25 of 500 clients" text

Reply:
Sounds good. It looks like the [pagination text "displaying X of X clients"](https://github.com/greenriver/hmis-frontend/blob/a028a949d9012265cd12393602bf0864d1b2466e/src/modules/dataFetching/components/GenericTableWithData.tsx#L385-L387) uses `paginationItemName` first and then `recordType`, rather than the other way round, so let's do that here too.

Reply:
yes! thanks